scheduler/src/main/java/org/apache/mesos/elasticsearch/scheduler/state/StatePath.java

```diff
@@ -43,4 +43,9 @@ public Boolean exists(String key) throws IOException {
         }
         return exists;
     }
+
+    public void rm(String key) throws IOException {
```

Comment:
It should be private. And I could probably inline now after removing the replace. And everyone understands what `rm` does, so a description would be a waste of lines of code.